torchci/scripts/check_alerts.py

```diff
@@ -221,12 +228,25 @@ def generate_no_flaky_tests_issue() -> Any:
     return issue
 
 
-def update_issue(issue: Dict, issue_number: int) -> Dict:
+def delete_old_comments(old_issue: Any) -> None:
+    # delete old comments from github bot
+    comments = old_issue["comments"]["nodes"]
+    for comment in comments:
+        if comment["bodyText"] == UPDATING_ALERT_COMMENT:
+            requests.delete(
+                f"https://api.github.com/repos/{REPO_OWNER}/{TEST_INFRA_REPO_NAME}/issues/comments/{comment['databaseId']}")
+
+
+def update_issue(issue: Dict, old_issue: Any) -> Dict:
```

Comment:
`old_issue` looks like a bug, I could only find one place where this function is called on line 422 and it passes only the existing issue number as the second parameter `update_issue(new_issue, existing_recurrent_job_failure_alert["number"])`

Reply:
fixed